tools/issue-labeler/src/IssueLabelerService/Program.cs

```diff
@@ -4,18 +4,20 @@
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using AzureRAGService;
 using Hubbup.MikLabelModel;
 
 var host = new HostBuilder()
```

Comment:
This is where we could do all the DI registration/creation.    Config can be tied into this as well via the AppConfig provider or whatever we choose.